.ci_support/linux_64_.yaml

```diff
@@ -6,3 +6,5 @@ channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
```

Comment:
Don't worry about this.  Your `set python_min` will override this.  It's just the system-wide default.